src/bundle-manifest.ts

```diff
@@ -0,0 +1,160 @@
+/**
+ * A bundle is a grouping of files which serve the need of one or more
+ * entrypoint files.
+ */
+export interface IBundle {
```

Comment:
I didn't realize that classes essentially worked like interfaces where type expressions were concerned.  Ditching the IBundle interface in favor of Bundle class.